src/data/structure/Tuple2.java

```diff
@@ -0,0 +1,26 @@
+package data.structure;
+
+/**
+ * Supports simple pairs of arbitrary items
+ * @author benjaminposerow
+ *
+ * @param <T>
+ * @param <U>
+ */
+public class Tuple2<T,U> {
```

Comment:
It doesn't look like there is. I was thinking Guava may have implemented, but it looks like they explicitly decided not to: https://github.com/google/guava/wiki/IdeaGraveyard